File: 1010.java
import java.util.*;
 
public class Main {
	
	public static void main(String [] args) {
		
		Scanner input = new Scanner(System.in);
		int budget, rest = 0, deposit = 0,  deadmonth = 0;
		int insufficiency = 0;
		
		for (int i = 1; i <= 12; i++) {
			rest = rest + 300;
			budget = input.nextInt();
			rest -= budget;
			if (rest >= 100) {
				deposit += rest - rest % 100;
				rest = rest % 100;
			}
			if (rest < 0) {
				insufficiency += 1;
				if (insufficiency == 1) deadmonth = i;
			}
		}
		input.close();
		if (insufficiency > 0) System.out.println("-" + deadmonth);
		else System.out.println((int) (rest + deposit * 1.2));
	}
}

File: 1010.cpp
#include <iostream>
int budget,rest,deposit,month,deadmonth;
int insufficiency;
int main(){
    using namespace std;
    insufficiency=0;
    for (month=1; month<=12; month++){
        rest+=300;
        cin>>budget;
        rest-=budget;
        if (rest>=100){
            deposit+=rest-rest%100;
            rest=rest%100;
        }
        if (rest<0){
            insufficiency++;
            if (insufficiency==1) deadmonth=month;
        }
    }
    if (insufficiency) cout<<"-"<<deadmonth;
        else cout<<rest+deposit*1.2;
    return 0;
}